extras/drift_postgres/lib/src/pg_database.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:drift/backends.dart';
+import 'package:postgres/postgres.dart';
+
+///
+class PgDatabase extends DelegatedDatabase {
+  ///
+  PgDatabase(PostgreSQLConnection connection)
+      : super(_PgDelegate(connection, connection),
+            isSequential: true, logStatements: true);
+
+  ///
+  factory PgDatabase.open(String host, int port, String databaseName,
+      {String? username,
+      String? password,
+      int timeoutInSeconds = 30,
+      int queryTimeoutInSeconds = 30,
+      String timeZone = 'UTC',
+      bool useSSL = false,
+      bool isUnixSocket = false}) {
+    return PgDatabase(PostgreSQLConnection(host, port, databaseName,
+        username: username,
+        password: password,
+        timeoutInSeconds: timeoutInSeconds,
+        queryTimeoutInSeconds: queryTimeoutInSeconds,
+        timeZone: timeZone,
+        useSSL: useSSL,
+        isUnixSocket: isUnixSocket));
+  }
+}
+
+///
```

Comment:
We don't need documentation comments for private classes :D But I can also fill out the docs later.